dap_job_quality/utils/prodigy_data_utils.py

```diff
@@ -0,0 +1,71 @@
+from spacy.tokens import Span, Doc
```

Comment:
nice! reads well. you could add a descritption of this file 
i.e.
"""
helper functions for prodigy-labelled data.
"""